entra-id-users-support-quickstart/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "entra-id-users-support-quickstart",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "@azure/communication-common": "^2.3.2-beta.1",
```

Comment:
Should we use the new stable version, or keep this beta?

Reply:
fixed